src/jit/emit.h

```diff
@@ -153,7 +153,7 @@ class emitLocation
     // A constructor for code that needs to call it explicitly.
     void Init()
     {
-        this->emitLocation::emitLocation();
+        *this = emitLocation();
```

Comment:
it looks like placement new is working. I'll change to placement new too to be consistent.

Reply:
I had compilation errors. Reverted to copy method instead.